app/services/metrc_service/plant/move.rb

```diff
@@ -39,121 +30,119 @@ def prior_move_transactions
+        return :move_plants if previous_growth_phase.include?('veg') && new_growth_phase.include?('veg')
 
-        return :move_plants if previous_zone.include?('flow') && new_zone.include?('flow')
+        return :move_plants if previous_growth_phase.include?('flow') && new_growth_phase.include?('flow')
 
-        return :change_plant_growth_phases if previous_zone.include?('veg') && new_zone.include?('flow')
+        return :change_plant_growth_phases if previous_growth_phase.include?('veg') && new_growth_phase.include?('flow')
 
         DEFAULT_MOVE_STEP
       end
 
-      def move_plants(options)
-        items   = get_items(options[:seeding_unit_id])
+      def move_plants
         payload = items.map do |item|
           {
             Id: nil,
             Label: item.relationships.dig('barcode', 'data', 'id'),
-            Location: options[:zone_name],
-            ActualDate: @attributes.dig('start_time')
+            Location: batch.zone.name,
+            ActualDate: start_time
           }
         end
 
         call_metrc(:move_plants, [payload])
       end
 
-      def move_plant_batches(options)
+      def move_plant_batches
         payload = {
           Name: batch_tag,
-          Location: options[:zone_name],
-          MoveDate: @attributes.dig('start_time')
+          Location: batch.zone.name,
+          MoveDate: start_time
         }
 
         call_metrc(:move_plant_batches, [payload])
       end
 
-      def change_growth_phase(options)
-        batch        = options[:batch]
-        items        = get_items(batch.seeding_unit.id)
+      def change_growth_phase
         first_tag_id = items.first.id
         barcode      = items.find { |item| item.id == first_tag_id }.relationships.dig('barcode', 'data', 'id')
-        growth_phase = batch.zone.sub_stage.name
 
         payload = {
           Name: batch_tag,
           Count: batch.quantity.to_i,
           StartingTag: barcode,
-          GrowthPhase: growth_phase,
-          NewLocation: options[:zone_name],
-          GrowthDate: @attributes.dig('start_time'),
+          GrowthPhase: normalized_growth_phase,
+          NewLocation: batch.zone.name,
+          GrowthDate: start_time,
           PatientLicenseNumber: nil
         }
 
         call_metrc(:change_growth_phase, [payload])
       end
 
-      def change_plant_growth_phases(options)
-        batch        = options[:batch]
-        seeding_unit = batch.zone.attributes['seeding_unit']
-        items        = get_items(options[:seeding_unit_id])
-
+      def change_plant_growth_phases
         payload = items.map do |item|
           {
             Id: nil,
             Label: item.relationships.dig('barcode', 'data', 'id'),
-            NewTag: seeding_unit['name'], # TODO: Fix me
-            GrowthPhase: normalize_growth_phase(seeding_unit['name']),
+            NewTag: '', # seeding_unit['name'], # TODO: Fix me
```

Comment:
@betacar Not sure what to do here. Is there a scenario where we change the tags on move between growth phases?

Reply:
No idea. This was here because it's on Metrc's docs. Please confirm with Erin.